_sass/jekyll-theme-architect.scss

```diff
@@ -563,77 +560,86 @@ span.requisites {
   }
   header {
     padding: 20px 0;
-  }
-  header .inner {
-    position: relative;
-  }
-  header h1, header h2 {
-    width: 100%;
-  }
-  header h1 {
-    font-size: 32px;
-  }
-  header h1:before {
-    content: "";
-    background: url(../images/Icon1_48_blue.png) no-repeat;
-    width: 48px;
-    height: 48px;
-    display: block;
-    float: left;
-    margin-top: -7px;
-  }
-  header h2 {
-    font-size: 24px;
-  }
-  header a.button {
-    position: relative;
-    display: inline-block;
-    width: auto;
-    height: auto;
-    padding: 5px 10px;
-    margin-top: 15px;
-    font-size: 13px;
-    line-height: 1;
-    color: #2879d0;
-    text-align: center;
-    background-color: #9ddcff;
-    background-image: none;
-    border-radius: 5px;
-    -moz-border-radius: 5px;
-    -webkit-border-radius: 5px;
-  }
-  header a.button small {
-    display: inline;
-    font-size: 13px;
-  }
-  #main-content,
-  aside#sidebar {
+
+    .inner {
+      position: relative;
+    }
+
+    h1, h2 {
+      width: 100%;
+    }
+
+    h1 {
+      font-size: 32px;
+
+      &:before  {
+        content: "";
+        background: url(../images/Icon1_48_blue.png) no-repeat;
+        width: 48px;
+        height: 48px;
+        display: block;
+        float: left;
+        margin-top: -7px;
+      }
+    }
+
+    h2 {
+      font-size: 24px;
+    }
+
+    a.button {
+      position: relative;
+      display: inline-block;
+      width: auto;
+      height: auto;
+      padding: 5px 10px;
+      margin-top: 15px;
+      font-size: 13px;
+      line-height: 1;
+      color: #2879d0;
+      text-align: center;
+      background-color: #9ddcff;
+      background-image: none;
+      border-radius: 5px;
+      -moz-border-radius: 5px;
+      -webkit-border-radius: 5px;
+
+      small {
+        display: inline;
+        font-size: 13px;
+      }
+    }
+  }
+  // remove aside from selector ***continue here***
```

Comment:
Oops I left this comment for myself because I started yesterday and finished today XD will remove